common/src/main/java/org/keycloak/common/util/Base64.java

```diff
@@ -733,7 +738,7 @@ public static String encodeBytes( byte[] source ) {
      * Example options:<pre>
      *   GZIP: gzip-compresses object before encoding it.
      *   DO_BREAK_LINES: break lines at 76 characters
-     *     <i>Note: Technically, this makes your encoding non-compliant.</i>
+     *     <i>Note: Technically, without line break your encoding may become non-compliant (see rfc2045 and rfc4648).</i>
```

Comment:
Unfortunately adding line breaks causes many SAML Service Providers to fail validating signatures. This is a blocking issue for people being able to upgrade Keycloak now.